test/ApplyArgs.purs

```diff
@@ -1,380 +0,0 @@
-module Test.Ctl.ApplyArgs (main, suite, contract) where
-
-import Contract.Prelude
-
-import Contract.Monad (Contract, launchAff_)
-import Contract.Numeric.BigNum as BigNum
```

Comment:
I see no reason for removing this test. It could work just fine.

We need the following setup:

- `ps-apply-args` included in CTL's `packages.dhall` and `spago.dhall` (so that this test could work)
- the corresponding NPM package included in both the template and the CTL itself
- `ps-apply-args` NOT included in template's `spago.dhall`, but included in template's `packages.dhall`. CTL's dependencies in [template's package.dhall](https://github.com/Plutonomicon/cardano-transaction-lib/blob/5bf278436b46d910ae4339add6942bf362ef4a82/templates/ctl-scaffold/packages.dhall#L414) should NOT include ps-apply args.


This gives us the following properties:

- CTL can be used without ps-apply-args
- the template does not pull ps-apply-args by default
- ps-apply-args can be easily added as a dependency into any project, including the template
- CTL has tests for ps-apply-args in its test suite